packages/search/src/Infrastructure/Symfony/HttpKernel/SuluSearchBundle.php

```diff
@@ -55,6 +55,11 @@ public function configure(DefinitionConfigurator $definition): void
                                     ->isRequired()
                                     ->cannotBeEmpty()
                                 ->end()
+                                ->arrayNode('resultToRouteName')
+                                    ->useAttributeAsKey('name')
+                                    ->scalarPrototype()
+                                    ->end()
```

Comment:
```suggestion
                                    ->scalarPrototype()->end()
```

from the docs: https://symfony.com/doc/current/components/config/definition.html#adding-node-definitions-to-the-tree